src/components/Characteristics.js

```diff
@@ -1,10 +1,14 @@
 import { Characteristic } from "./Characteristic";
 
 export const Characteristics = ({ characteristics }) => {
-  const characteristicBubbles = Object.keys(characteristics).map(characteristic => {
-    const score = characteristics[characteristic];
-    return <Characteristic key={characteristic} characteristic={characteristic} score={score} />
-  });
+  const characteristicBubbles = [
+    <Characteristic key='brawn' characteristic='brawn' score={characteristics.brawn} />,
```

Comment:
Yanno, now that I think about it, I bet we could do what you were thinking and map through the keys. It helped to see it in this format.